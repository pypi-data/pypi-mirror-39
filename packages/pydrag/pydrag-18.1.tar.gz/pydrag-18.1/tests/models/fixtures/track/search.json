{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.1"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&limit=5&method=track.search&page=4&track=gun"
            },
            "response": {
                "body": {
                    "string": "{\"results\":{\"opensearch:Query\":{\"#text\":\"\",\"role\":\"request\",\"startPage\":\"4\"},\"opensearch:totalResults\":\"1443087\",\"opensearch:startIndex\":\"15\",\"opensearch:itemsPerPage\":\"5\",\"trackmatches\":{\"track\":[{\"name\":\"Welcome to the Jungle\",\"artist\":\"Guns N' Roses\",\"url\":\"https://www.last.fm/music/Guns+N%27+Roses/_/Welcome+to+the+Jungle\",\"streamable\":\"FIXME\",\"listeners\":\"1231068\",\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"mbid\":\"\"},{\"name\":\"Guns and Roses\",\"artist\":\"Lana Del Rey\",\"url\":\"https://www.last.fm/music/Lana+Del+Rey/_/Guns+and+Roses\",\"streamable\":\"FIXME\",\"listeners\":\"129126\",\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"mbid\":\"CENSORED545d\"},{\"name\":\"Under the Gun\",\"artist\":\"The Killers\",\"url\":\"https://www.last.fm/music/The+Killers/_/Under+the+Gun\",\"streamable\":\"FIXME\",\"listeners\":\"340744\",\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"mbid\":\"CENSOREDe4f1\"},{\"name\":\"Paradise City\",\"artist\":\"Guns N' Roses\",\"url\":\"https://www.last.fm/music/Guns+N%27+Roses/_/Paradise+City\",\"streamable\":\"FIXME\",\"listeners\":\"1092670\",\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"mbid\":\"CENSORED18d6\"},{\"name\":\"Gun-Shy\",\"artist\":\"Grizzly Bear\",\"url\":\"https://www.last.fm/music/Grizzly+Bear/_/Gun-Shy\",\"streamable\":\"FIXME\",\"listeners\":\"164751\",\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"mbid\":\"CENSORED8564\"}]},\"@attr\":{}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "3414"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sun, 11 Nov 2018 19:47:42 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
