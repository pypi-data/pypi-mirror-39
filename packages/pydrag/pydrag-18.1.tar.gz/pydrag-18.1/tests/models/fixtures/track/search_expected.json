{
    "data": [
        {
            "artist": {
                "name": "Guns N' Roses"
            },
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 1231068,
            "mbid": "",
            "name": "Welcome to the Jungle",
            "url": "https://www.last.fm/music/Guns+N%27+Roses/_/Welcome+to+the+Jungle"
        },
        {
            "artist": {
                "name": "Lana Del Rey"
            },
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 129126,
            "mbid": "CENSORED545d",
            "name": "Guns and Roses",
            "url": "https://www.last.fm/music/Lana+Del+Rey/_/Guns+and+Roses"
        },
        {
            "artist": {
                "name": "The Killers"
            },
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 340744,
            "mbid": "CENSOREDe4f1",
            "name": "Under the Gun",
            "url": "https://www.last.fm/music/The+Killers/_/Under+the+Gun"
        },
        {
            "artist": {
                "name": "Guns N' Roses"
            },
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 1092670,
            "mbid": "CENSORED18d6",
            "name": "Paradise City",
            "url": "https://www.last.fm/music/Guns+N%27+Roses/_/Paradise+City"
        },
        {
            "artist": {
                "name": "Grizzly Bear"
            },
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 164751,
            "mbid": "CENSORED8564",
            "name": "Gun-Shy",
            "url": "https://www.last.fm/music/Grizzly+Bear/_/Gun-Shy"
        }
    ],
    "limit": 5,
    "page": 4,
    "total": 1443087
}
